speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DeveloperMenu.cpp

```diff
@@ -265,6 +276,43 @@ static void SetSteerThreshold(void*)
     SetThreshold(m_decisionThresholds.Steer, m_steerThresholdControl);
 }
 
+/// @breif set the default values of threshold boxes based on the InterventionType.
+static void SetDefaultValues(void*)
```

Comment:
SetDefaultValues implies setting default values for the entire menu. Rename to e.g. SetDefaultThresholdValues

Reply:
made defines